src/config.ts

```diff
@@ -31,6 +31,7 @@ export const config = rt
     workDir: rt.String,
     commentOnPr: rt.Boolean,
     options: options,
+    pulumiVersion: rt.String,
```

Comment:
I don't think we even need to expose this - I think what we should do is internally set this to match 3.x and make sure we always install the new 3.x version

Allowing a user to control this may still cause problems for us with versions being mismatched

A CLI version will have no real bearing on the program itself

Reply:
This is not a required field, and it defaults to `^3` now.

We could make sure the version set by the user qualifies (the range that works with a particular version of the GHA). We probably don't need that if we're only doing this internally.

Reply:
we are going to be doing this internally so that users don't care about this anymore